<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorClass</class>
 <widget class="QMainWindow" name="CalculatorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="button7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid grey;
	background-color: Azure;
}


QPushButton:hover:pressed{
	border: 1px solid grey;
	background-color: PowderBlue;
}

QPushButton:hover{
	border: 1px solid grey;
	background-color: AliceBlue;
}

</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid grey;
	background-color: Azure;
}


QPushButton:hover:pressed{
	border: 1px solid grey;
	background-color: PowderBlue;
}

QPushButton:hover{
	border: 1px solid grey;
	background-color: AliceBlue;
}

</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid grey;
	background-color: Azure;
}


QPushButton:hover:pressed{
	border: 1px solid grey;
	background-color: PowderBlue;
}

QPushButton:hover{
	border: 1px solid grey;
	background-color: AliceBlue;
}

</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid grey;
	background-color: Gainsboro;
}



QPushButton:hover:pressed{
	border: 1px solid grey;
	background-color: Gray;
}

QPushButton:hover{
	border: 1px solid grey;
	background-color: DarkGrey;
}

</string>
    </property>
    <property name="text">
     <string>M+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMR">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid grey;
	background-color: Gainsboro;
}



QPushButton:hover:pressed{
	border: 1px solid grey;
	background-color: Gray;
}

QPushButton:hover{
	border: 1px solid grey;
	background-color: DarkGrey;
}

</string>
    </property>
    <property name="text">
     <string>MR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDiv">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid grey;
	background-color: Gainsboro;
}



QPushButton:hover:pressed{
	border: 1px solid grey;
	background-color: Gray;
}

QPushButton:hover{
	border: 1px solid grey;
	background-color: DarkGrey;
}

</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid grey;
	background-color: Azure;
}


QPushButton:hover:pressed{
	border: 1px solid grey;
	background-color: PowderBlue;
}

QPushButton:hover{
	border: 1px solid grey;
	background-color: AliceBlue;
}

</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid grey;
	background-color: Azure;
}


QPushButton:hover:pressed{
	border: 1px solid grey;
	background-color: PowderBlue;
}

QPushButton:hover{
	border: 1px solid grey;
	background-color: AliceBlue;
}

</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMC">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid grey;
	background-color: Gainsboro;
}



QPushButton:hover:pressed{
	border: 1px solid grey;
	background-color: Gray;
}

QPushButton:hover{
	border: 1px solid grey;
	background-color: DarkGrey;
}

</string>
    </property>
    <property name="text">
     <string>MC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid grey;
	background-color: Azure;
}


QPushButton:hover:pressed{
	border: 1px solid grey;
	background-color: PowderBlue;
}

QPushButton:hover{
	border: 1px solid grey;
	background-color: AliceBlue;
}

</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSub">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid grey;
	background-color: Gainsboro;
}



QPushButton:hover:pressed{
	border: 1px solid grey;
	background-color: Gray;
}

QPushButton:hover{
	border: 1px solid grey;
	background-color: DarkGrey;
}

</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid grey;
	background-color: Azure;
}


QPushButton:hover:pressed{
	border: 1px solid grey;
	background-color: PowderBlue;
}

QPushButton:hover{
	border: 1px solid grey;
	background-color: AliceBlue;
}

</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid grey;
	background-color: Azure;
}


QPushButton:hover:pressed{
	border: 1px solid grey;
	background-color: PowderBlue;
}

QPushButton:hover{
	border: 1px solid grey;
	background-color: AliceBlue;
}

</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMulti">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid grey;
	background-color: Gainsboro;
}



QPushButton:hover:pressed{
	border: 1px solid grey;
	background-color: Gray;
}

QPushButton:hover{
	border: 1px solid grey;
	background-color: DarkGrey;
}

</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid grey;
	background-color: Azure;
}


QPushButton:hover:pressed{
	border: 1px solid grey;
	background-color: PowderBlue;
}

QPushButton:hover{
	border: 1px solid grey;
	background-color: AliceBlue;
}

</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSum">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid grey;
	background-color: Gainsboro;
}



QPushButton:hover:pressed{
	border: 1px solid grey;
	background-color: Gray;
}

QPushButton:hover{
	border: 1px solid grey;
	background-color: DarkGrey;
}

</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClear">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid grey;
	background-color: LightCyan;
}


QPushButton:hover:pressed{
	border: 1px solid grey;
	background-color: Aquamarine;
}

QPushButton:hover{
	border: 1px solid grey;
	background-color: PaleTurquoise;
}

</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button0">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid grey;
	background-color: Azure;
}


QPushButton:hover:pressed{
	border: 1px solid grey;
	background-color: PowderBlue;
}

QPushButton:hover{
	border: 1px solid grey;
	background-color: AliceBlue;
}

</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonEqual">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid grey;
	background-color: Silver;
}



QPushButton:hover:pressed{
	border: 1px solid grey;
	background-color: LightSlateGray;
}

QPushButton:hover{
	border: 1px solid grey;
	background-color: DarkGrey;
}


</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSign">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid grey;
	background-color: LightCyan;
}


QPushButton:hover:pressed{
	border: 1px solid grey;
	background-color: Aquamarine;
}

QPushButton:hover{
	border: 1px solid grey;
	background-color: PaleTurquoise;
}

</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Display">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Calculator.qrc"/>
 </resources>
 <connections/>
</ui>
